exe/dual_run

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env ruby
+
+require "pty"
+
+command = ARGV.join(" ")
+
+if command == ""
+  puts "No command specified"
+  exit 1
+end
+
+puts "Running in parallel:"
+puts "BUNDLE_GEMFILE=Gemfile.next #{command}"
+puts command
```

Comment:
Probably we don't need to print the command twice, it was printed in the line before, what about to print it on each block.

Like so:

```
Running in parallel:

next results
BUNDLE_GEMFILE=Gemfile.next bundle exec rspec spec/models/
================================================================================

Randomized with seed 41263
...................

Finished in 0.25773 seconds (files took 4.02 seconds to load)
19 examples, 0 failures

Randomized with seed 41263




(waiting for current command to finish)
current results
BUNDLE_GEMFILE=Gemfile bundle exec rspec spec/models/
================================================================================

Randomized with seed 40666
...................

Finished in 0.23505 seconds (files took 3.96 seconds to load)
19 examples, 0 failures

Randomized with seed 40666
```